### PR TITLE
Fix the RHEL docker build

### PR DESCRIPTION
**Pull request by dlbock**
Changes:
- Install `docker` by adding docker repos to `/etc/yum.repos.d/docker.repo` and installing via `microdnf`
- Install a specific slim version of `gomplate` via `curl` from github
- Remove the installation of `iptables-services` as that is failing right now. This is only needed for service mesh support. Will have to rework this at a later time. We need to update the `instana/agent` image on the RedHat registry as this is holding up the `instana-agent-operator` image update, which has a security vulnerability.
- Uncomment the section in `Jenkinsfile` to build the RHEL image and publish to the internal registry

### CODE REVIEW

**Comment by dlbock** [comment]
I have no credentials to upload a new version of the RHEL image to the RH registry. We either need to reenable the Jenkins build to build a new version of the image, or someone can manually build it and upload it.

File: Jenkinsfile [hunk 1]
```diff
@@ -27,13 +27,12 @@ node {
...
   }
-  
+
   stage ('Build RHEL Agent Docker Image') {
-    //TODO fix redhat build
-    //if (env.RHEL.toBoolean()) {
-    //  println "Building rhel agent docker image"
-    //  buildImage(RHEL_IMAGE_NAME, "rhel", releaseVersion, null)
-    //}
+    if (env.RHEL.toBoolean()) {
```
Comment (htreu) [inline Jenkinsfile]: do we still need this safe guard? I suggest also building the REHL image every time we build the "normal" images.

File: Jenkinsfile [hunk 2]
```diff
@@ -45,20 +44,19 @@ node {
...
       println "Push static agent docker image to prerelease"
       publishImage(STATIC_IMAGE_NAME, "${DOCKER_REGISTRY_INTERNAL}/instana/prerelease/agent/static", releaseVersion)
-      
-      //TODO fix redhat build
-      //if (env.RHEL.toBoolean()) {
-      //  println "Push rhel agent docker image to prerelease"
-      //  publishImage(RHEL_IMAGE_NAME, "${DOCKER_REGISTRY_INTERNAL}/instana/prerelease/agent/rhel", releaseVersion)
-      //}
+
+      if (env.RHEL.toBoolean()) {
```
Comment (htreu) [inline Jenkinsfile]: same here

File: Jenkinsfile (hunk 1 above)
Comment (spanneberg) [reply Jenkinsfile]: would also handle it the same way

Comment (dlbock) [reply Jenkinsfile]: Alrighty, I'll remove the safeguard, just a moment

**Pull request merged**